test/e2e/v1beta1/scripts/gh-actions/setup-katib.sh

```diff
@@ -68,6 +84,7 @@ kubectl -n kubeflow get pod
 kubectl apply -f ../../testdata/valid-experiment.yaml
 kubectl delete -f ../../testdata/valid-experiment.yaml
 
+sleep 60
```

Comment:
@tenzen-y Thanks for your recommendation :pray: 

With my local environments, I figure out validatingwebhook sometimes works when I try to create experiment right-after katib-controller pod became ready. (even though after pod's status became ready)

I guess this makes sense since validating webhook uses katib-controller's API (/validate-experiment), and if katib-controller's readiness probe does not meaning it works at all.

So I just add time sleep here, and it seems like works well even with github action. (maybe some slow k8s environments rather than my local machine)
https://github.com/kubeflow/katib/runs/7807939711?check_suite_focus=true

Reply:
@anencore94 I got it! Thanks for your investigation!

I guess that katib-controller spends a bit long time completing setup since the GitHub-hosted runner is a very tight computing resource for our heavy integration tests.

Reply:
I guess so too. I just add some comments on this shell script. and it is ready for review now. Thanks :)